client/src/components/Dialog.js

```diff
@@ -20,6 +20,7 @@ const Dialog = ({ visible, title, onClose, children }) => {
           <div // eslint-disable-line jsx-a11y/no-static-element-interactions
             onClick={onClose}
             className={css.close}
+            hidden={hideCloseSymbol}
```

Comment:
I'd probably just not use the corner prop of Card if the symbol was to be hidden. If you're effectively removing it on a true/false anyways, might as well not throw the div into the corner.